docs/pages/server-access/guides/openssh.mdx

```diff
@@ -10,7 +10,13 @@ accept SSH certificates dynamically issued by a Teleport CA.
 
 Using Teleport and OpenSSH has the advantage of getting you up
 and running, but in the long run, we would recommend replacing `sshd` with `teleport`.
-We've outlined these reasons in [OpenSSH vs Teleport SSH for Servers?](https://goteleport.com/blog/openssh-vs-teleport/)
+`teleport` SSH servers have support for multiple features that are incompatible with OpenSSH:
```

Comment:
Looking at this list, I think we should choose three features and add a sentence to describe each one. Otherwise, it's not really possible to understand what these terms describe without clicking on the links.

Reply:
I kept all of them, I think it's important for users to know what agentless nodes can and can't do compared to Teleport nodes. If you still think the list should be shortened I can remove some though.